sim_pipeline/gg_lens_pop.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, lens_type='early-type', source_type='galaxies', kwargs_deflec
             kwargs_deflector_cut = {}
         if kwargs_mass2light is None:
             kwargs_mass2light = {}
+        if cosmo is None:
+            from astropy.cosmology import FlatLambdaCDM
```

Comment:
This is great! Perhaps make a warning in the code when executing the lines without providing a cosmo instance. i.e.
Warning("No cosmology provided, instead uses flat LCDM with default parameters")